runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java

```diff
@@ -518,6 +529,46 @@ public void onGameTick(GameTick event)
 		Player player = client.getLocalPlayer();
 		WorldPoint currentWorldPoint = player.getWorldLocation();
 
+		SkullIcon currentSkullIcon = player.getSkullIcon();
+		int currentRegionID = WorldPoint.fromLocalInstance(client, client.getLocalPlayer().getLocalLocation()).getRegionID();
+		final int abyssRegionID = 12107;
```

Comment:
should probably be a private static final int at the start of the file so it's more clearly a constant definition

Reply:
I will use your first suggestion, as to my knowledge there isn't a regionID api of any sort yet.